invenio/templates/_helpers.tpl

```diff
@@ -2,9 +2,58 @@
 Get the redis host name.
 */}}
 {{- define "redis.host_name" -}}
-{{- if .Values.redis.host }}
-    {{- printf "%s" (tpl .Values.redis.host $) -}}
+{{- if not (.Values.redis.invenio.enabled) }}
+    {{- if .Values.redis.invenio.host}}
+        {{- printf "%s" (tpl .Values.redis.invenio.host $) -}}
```

Comment:
moderate: what about defaulting `redis.invenio.host` to `cache` instead of empty string (`""`) and calculating it here? we would avoid one if/else. Same applies for `search`,`db`, and `mq` below. We are actually saying that it defaults to `cache` in the README.

Reply:
The way it is the solution prevents someone from deleting the value(s) from `values.yaml`. It's defined in every possible case.